main_window/competition/comp_app.py

```diff
@@ -186,6 +172,22 @@ def setup_subwindow(self):
 
         return sub
 
+    def setup_view_menu(self) -> None:
+        # Menubar for choosing rocket device view
+        if len(self.rocket_profile.mapping_devices) > 1:
+            view_menu = self.menuBar().children()[2]
+            self.map_view_menu = view_menu.addMenu("Map")
+
+            view_all = QAction("All", self)
+            all_devices = self.rocket_profile.mapping_devices
+            view_all.triggered.connect(lambda i, all_devices = all_devices: self.set_view_device(all_devices))
+            self.map_view_menu.addAction(view_all)
+
+        for device in self.rocket_profile.mapping_devices:
+            view_device = QAction(f'{device}', self)
+            view_device.triggered.connect(lambda i, device=device: self.set_view_device([device]))
+            self.map_view_menu.addAction(view_device)
```

Comment:
Looks like integration tests are failing for non-tantalus profiles.
```
>           self.map_view_menu.addAction(view_device)
E           AttributeError: 'CompApp' object has no attribute 'map_view_menu'

main_window/competition/comp_app.py:189: AttributeError
```
I think its because the indentation changed for this for loop.